### ISSUE TITLE
[BUG] Doesn't seem to work. (doesn't start)

### ISSUE THREAD

**Issue by masterofobzene** [Jules-WinnfieldX/CyberDropDownloader#125]
Running main.py with or without the bat file (windows) just opens and closes a cmd.
The script is failing somewhere.
Also, can it be installed as a portable script? (meaning outside of the python folder) I see you run a pip install while using the bat file. 
All requirements are installed.
![imagen](https://user-images.githubusercontent.com/94469116/166121668-748063bc-821a-4722-8497-a9758a434536.png)
Using the latest release, downloaded today 04/30.
Using Python 3.8.2

What's this line doing?

  File "....\main.py", line 9, in <module>
    from . import __version__ as VERSION  <-- failing here

Also, I only get these files after pip install
![imagen](https://user-images.githubusercontent.com/94469116/166121715-436b51e8-5794-4850-9c2f-8a7fcb804ffc.png)

I was using version 2.2.1 but it started to download corrupt files so I tried to upgrade and now the script doesn't work anymore.
Thanks.

**Comment by Jules-WinnfieldX** [comment]
Where to begin with this.. 

For help troubleshooting using the standard method of running, using the bat file provided or through calling cyberdrop-dl from the command line, I'd need to see the downloader.log file to tell you anything.

You could download the source code and do whatever you want with it, but I do not support a portable script version currently. 

`from . import version as VERSION` <- this line is simply importing the version number from the init file. 

Those are the site packages that should be there after the pip install. You'll also find a cyberdrop-dl.exe executable in the scripts folder. Which is what runs when calling cyberdrop-dl from the command line.

**Comment by Jules-WinnfieldX** [comment]
Running main.py does nothing because it isn't supposed to do anything when calling it directly.

**Comment by Jules-WinnfieldX** [comment]
To be clear. There are new start bat scripts that you need, in the releases section. The old ones won't work.

**Comment by masterofobzene** [comment]
Fixed. The old version worked with main.py that's why I kept calling it directly. I was also able to make the script work in "portable" mode with the old version too, so everything seemed to make sense that way.

Deleted everything, started over from scratch using the release, then just edited the batch file to make it easier for me.

```
rem pip install --upgrade cyberdrop-dl
.\URLs.txt
cyberdrop-dl
rem pause
```
Seems I missed some versions in the way, so many changes.

Thanks.
